docs/pages/custom-builds/schema.mdx

```diff
@@ -570,6 +570,100 @@ EAS provides a set of built-in reusable functions that you can use in a workflow
 
 > **info** **Tip:** Any function that is built-in and provided by EAS must start with the `eas/` prefix.
 
+#### `eas/maestro_test`
+
+All-in-one function that installs Maestro, prepares a testing environment (iOS Simulator or Android Emulator) and tests the app.
```

Comment:
```suggestion
All-in-one function that installs Maestro, prepares a testing environment (Android Emulator or iOS Simulator ), and tests the app.
```

When we reference Android, iOS, Web as platforms, we prefer in this order [as per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web)